.travis.yml

```diff
@@ -173,6 +173,7 @@ before_script:
           fi;
       fi
     - ./configdata.pm --dump
+    - export HARNESS_JOBS=${HARNESS_JOBS:-4}
```

Comment:
@DDvO this should make it the default for all Travis builds (in theory)

Reply:
The thought is that individual targets can override this default by defining a different HARNESS_JOBS value in their env property

Reply:
>  this should make it the default for all Travis builds (in theory)

Excellent.